docs/visual-testing/_partials/_fullpage-js.md

```diff
@@ -23,4 +38,53 @@ await browser.sauceVisualCheck('Long content page', {
 });
 ```
 
-<FullPageLimit />
+#### Mobile Native (beta)
+
+Options:
+
+- `delayAfterScrollMs`: Delay in ms after scrolling and before taking screenshots. The default value is 0. We recommend using this option for lazy loading content.
+- `nativeClipSelector`: Selector used to identify the first element to which clipping will be applied.
+- `scrollElement`: Scrollable element used for scrolling. The default is root element.
+- `scrollLimit`: Limit the number of screenshots taken for scrolling and stitching. The default value is 10. The value needs to be between 1 and 10.
+
+:::note
+It is recommended to define the `scrollElement` as the appropriate scrollable container.
+:::
+
+```ts
+await browser.sauceVisualCheck('Long content page', {
+  // Enable full page screenshot and customize the behavior
+  fullPage: {
+    scrollElement: $('//XCUIElementTypeCollectionView'),
```

Comment:
can we also show an example with android? I know its the same but I know our users will struggle with selectors